exec/java-exec/src/main/java/org/apache/drill/exec/planner/logical/DrillReduceAggregatesRule.java

```diff
@@ -100,8 +108,13 @@ public void onMatch(RelOptRuleCall ruleCall) {
    */
   private boolean containsAvgStddevVarCall(List<AggregateCall> aggCallList) {
     for (AggregateCall call : aggCallList) {
-      if (call.getAggregation() instanceof SqlAvgAggFunction
-          || call.getAggregation() instanceof SqlSumAggFunction) {
+      SqlAggFunction sqlAggFunction = call.getAggregation();
+      if(sqlAggFunction instanceof DrillCalciteSqlWrapper) {
```

Comment:
There are a bunch of unwrap calls like this. Is there a way to avoid this? Otherwise, devs need to aware of this unwrapping for *all* Calcite (SQL standard) functions.